include/deal.II/base/utilities.h

```diff
@@ -178,6 +178,14 @@ namespace Utilities
   unsigned int
   needed_digits(const unsigned int max_number);
 
+  /**
+   * This function allows to cutoff a floating point number @p number
+   * after @p n_digits of accuracy.
```

Comment:
Can you please add a small example of what happens in that case. Furthermore, it would be good to state what happens for negative numbers - are they always truncated to zero or rounded down?